VideoGame_Beginning

```diff
@@ -0,0 +1,17 @@
+Resaons for Argument
+
+Dataset
+
+Questions
+
+Theory
+
+Explanation
+
+Thesis
```

Comment:
How does critic score relate to global players?

How does User Score relate to/increase US game sales?

Which video game maker benefits the most by critic and user scores?